### PR TITLE
Core functionality done

### PR DESCRIPTION
**Pull request by kivistikliis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,6 +129,8 @@ Errors:
 
 DELETE **/events/<EVENT_ID>?token=<ACCESS_TOKEN>** </br>
 Archives an event with the given event ID. Used instead of deletion to keep the data. Returns the changed event.
+</br> Authentication-service: verifies permission.
```
Comment (dee-me-tree-or-love) [inline README.md:132]: @dmarkov00 you might want to take that into account! 😁

File: controllers/db_event_operations.py [hunk 2]
```diff
@@ -64,22 +79,31 @@ def update(self, id, field, value, token):
...
         nr_participants = len(participants)
         self.events_col.update_one({'_id': ObjectId(event_id)}, {"$set": {"volunteers": nr_participants}}, upsert=True)
+        self.update_search_service('volunteers', nr_participants, id)
         return
 
-
+    # add updated event to search service
+    def update_search_service(self, field, value, event_id):
+        add_to_service = add_updated_event_to_search(field, value, event_id)
+        if add_to_service.status_code > 299:
```
Comment (dee-me-tree-or-love) [inline controllers/db_event_operations.py:107]: Ha, I like this status check 😁👍

File: controllers/db_participation_operations.py [hunk 3]
```diff
@@ -88,3 +90,15 @@ def verify_if_arrived(self, event_id, user_id):
         result = {'enrolled': bool(joined), 'arrived': bool(appeared)}
         return to_json(result)
 
+
+# external method to call user service
+    def give_points(self, event_id, user_id):
+        event = self.events_col.find_one({"_id": ObjectId(event_id)})
+        points = event['points']
+        success = give_influence_points(points, user_id)
```
Comment (dee-me-tree-or-love) [inline controllers/db_participation_operations.py:98]: Btw, I have updated the user-service to also include actual user_id, which is created upon registration of the user... 
Was bad not to in the first place - sorry. 
Now either of the ways should work - if you want to get the user by `username` or by `user_id`
From the readme: https://github.com/Volunteero/user-service/blob/master/README.md
> PUT: ../altar/v1/users/find
purpose: find a user by request
body in: (request)  
`{
	"username":"dafoe5"  
}`
or
`{
	"user_id":"dafoe5"  
}`
note: if the input body contains both user_id and username, the preference is for the latter

Sorry for the inconvenience...

**Review by dee-me-tree-or-love — APPROVED** [review]
Looks pretty good!

File: controllers/db_participation_operations.py (hunk 3 above)
Comment (kivistikliis) [reply controllers/db_participation_operations.py:98]: Ah its good no worries! I think the username/id went by url anyway? Or it  changed, I will check

**Pull request merged**